charts/lnd/templates/export-secrets-configmap.yaml

```diff
@@ -23,3 +23,7 @@ data:
     export MACAROON=$(base64 ./macaroon/admin.macaroon | tr -d '\n\r')
     kubectl create secret generic {{ include "lnd.fullname" . }}-macaroon --from-literal=admin.macaroon=$MACAROON --dry-run=client -o yaml | kubectl apply -f -
     kubectl create secret generic {{ include "lnd.fullname" . }}-all-macaroons --from-file=./macaroon --dry-run=client -o yaml | kubectl apply -f -
+    
+    MACAROON_HEADER="Grpc-Metadata-macaroon: $(xxd -ps -u -c 1000 ./macaroon/admin.macaroon)" 
+    export PUBKEY=$(curl -s --cacert ./tls.cert --header "$MACAROON_HEADER"  https://{{ include "lnd.fullname" . }}:8080/v1/getinfo | jq .identity_pubkey)
```

Comment:
Any reason for not using `kubectl exec` to get the pubkey? That command would look simpler and be a one-liner I think